cloud_platform_ovh/models/cloud_platform.py

```diff
@@ -27,15 +30,85 @@ def _platform_kinds(self):
 
     @api.model
     def _config_by_server_env_for_ovh(self):
+        fs_kinds = self._filestore_kinds()
         configs = {
-            'prod': PlatformConfig(filestore=FilestoreKind.swift),
-            'integration': PlatformConfig(filestore=FilestoreKind.swift),
-            'labs': PlatformConfig(filestore=FilestoreKind.swift),
-            'test': PlatformConfig(filestore=FilestoreKind.db),
-            'dev': PlatformConfig(filestore=FilestoreKind.db),
+            'prod': PlatformConfig(filestore=fs_kinds['swift']),
+            'integration': PlatformConfig(filestore=fs_kinds['swift']),
+            'labs': PlatformConfig(filestore=fs_kinds['swift']),
+            'test': PlatformConfig(filestore=fs_kinds['db']),
+            'dev': PlatformConfig(filestore=fs_kinds['db']),
         }
         return configs
 
     @api.model
-    def install_ovh(self):
-        self.install('ovh')
+    def _check_filestore(self, environment_name):
+        params = self.env['ir.config_parameter'].sudo()
+        use_swift = (params.get_param('ir_attachment.location') ==
+                     SWIFT_STORE_KIND.name)
+        if environment_name in ('prod', 'integration'):
+            # Labs instances use swift or s3 by default, but we don't want
```

Comment:
```suggestion
            # Labs instances use swift by default, but we don't want
```
As we are into the OVH module, we will use only swift, correct?